### PR TITLE
Alphabetize traits list

### PR DESCRIPTION
**Pull request by TikaelSol**

### CODE REVIEW

File: static/lang/en.json [hunk 1]
```diff
@@ -3932,7 +3934,6 @@
         "TraitDescriptionWitch": "This indicates abilities from the witch class.",
         "TraitDescriptionWizard": "This indicates abilities from the wizard class.",
         "TraitDescriptionWood": "Effects with the wood trait conjure or manipulate wood. Those that manipulate wood have no effect in an area without wood. Creatures with this trait consist primarily of wood or have a connection to magical wood.",
-        "TraitBottledBreath": "Bottled Breath",
         "TraitCenser": "Censer",
```
Comment (simonward) [inline static/lang/en.json:3937]: TraitCenser too.

Comment (TikaelSol) [reply static/lang/en.json:3937]: Damn, knew I was going to forget one. We can save that for a later PR though, mostly it's the riders on this that I want in.

File: src/scripts/config/index.ts [hunk 2]
```diff
@@ -1113,6 +1113,7 @@ export const PF2ECONFIG = {
         ocotan: "PF2E.LanguageOcotan",
         okaiyan: "PF2E.LanguageOkaiyan",
         osiriani: "PF2E.LanguageOsiriani",
+        orvian: "PF2E.LanguageOrvian",
```
Comment (stwlam) [inline src/scripts/config/index.ts:1116]: :ezren_wow:

Comment (stwlam) [inline src/scripts/config/index.ts:1116]:
```suggestion
        orvian: "PF2E.LanguageOrvian",
        osiriani: "PF2E.LanguageOsiriani",
```

File: src/scripts/config/index.ts [hunk 3]
```diff
@@ -1113,6 +1113,7 @@ export const PF2ECONFIG = {
...
         petran: "PF2E.LanguagePetran",
```
Comment (TikaelSol) [inline src/scripts/config/index.ts:1117]:
```suggestion
        okaiyan: "PF2E.LanguageOkaiyan",
        orvian: "PF2E.LanguageOrvian",
        osiriani: "PF2E.LanguageOsiriani",
        petran: "PF2E.LanguagePetran",
```

File: static/lang/en.json [hunk 4]
```diff
@@ -2198,6 +2198,7 @@
         "LanguageOkaiyan": "Okaiyan",
         "LanguageOrcish": "Orcish",
         "LanguageOsiriani": "Osiriani",
+        "LanguageOrvian": "Orvian",
```
Comment (stwlam) [inline static/lang/en.json:2201]:
```suggestion
        "LanguageOrvian": "Orvian",
        "LanguageOsiriani": "Osiriani",
```

File: static/lang/en.json [hunk 5]
```diff
@@ -2198,6 +2198,7 @@
...
         "LanguagePetran": "Petran",
```
Comment (TikaelSol) [inline static/lang/en.json:2202]:
```suggestion
        "LanguageOrvian": "Orvian",
        "LanguageOsiriani": "Osiriani",
        "LanguagePetran": "Petran",
```

**Pull request merged**